src/Features/LanguageServer/Protocol/Features/Options/BlockStructureOptionsStorage.cs

```diff
@@ -63,6 +64,9 @@ public static BlockStructureOptions GetBlockStructureOptions(this IGlobalOptionS
     public static readonly PerLanguageOption2<bool> CollapseMetadataImplementationsWhenFirstOpened = new(
         FeatureName, "CollapseMetadataImplementationsWhenFirstOpened", BlockStructureOptions.Default.CollapseMetadataImplementationsWhenFirstOpened,
         storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.CollapseMetadataImplementationsWhenFirstOpened"));
+    public static readonly PerLanguageOption2<bool> CollapseEmptyMetadataImplementationsWhenFirstOpened = new(
+        FeatureName, "CollapseEmptyMetadataImplementationsWhenFirstOpened", BlockStructureOptions.Default.CollapseEmptyMetadataImplementationsWhenFirstOpened,
+        storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.CollapseEmptyMetadataImplementationsWhenFirstOpened"));
```

Comment:
we don't often doc our options, but it might be worthwhile here to to distinguish what amkes this different from teh above.

Reply:
Changed these names so I think they're self explanatory now, but can always follow up if you disagree